McEngine/src/GUI/CBaseUIElement.cpp

```diff
@@ -1,147 +1,149 @@
-//================ Copyright (c) 2013, PG, All rights reserved. =================//
```

Comment:
Why do both CBaseUIElement.h and CBaseUIElement.cpp show up as being changed _completely_?
Try to find out what caused this and revert it, it makes it a bit hard to see any differences.